CMakeLists.txt

```diff
@@ -239,7 +241,22 @@ if (USE_BOOST)
   find_package (Boost 1.46 ${BOOST_REQUIRED_ARG})
   if (Boost_FOUND)
     set (OPENTURNS_HAVE_BOOST TRUE)
-
+    if (USE_MPFR)
+      find_package (MPFR)
+      if (MPFR_FOUND)
+        set (OPENTURNS_HAVE_MPFR TRUE)
+        list (APPEND OPENTURNS_PRIVATE_INCLUDE_DIRS ${MPFR_INCLUDE_DIRS})
+        list (APPEND OPENTURNS_PRIVATE_LIBRARIES ${MPFR_LIBRARIES})
+      endif ()
+    endif ()
+    if (USE_MPC)
+      find_package (MPC)
+      if (MPC_FOUND)
+        set (OPENTURNS_HAVE_MPC TRUE)
+        list (APPEND OPENTURNS_PRIVATE_INCLUDE_DIRS ${MPC_INCLUDE_DIRS})
+        list (APPEND OPENTURNS_PRIVATE_LIBRARIES ${MPC_LIBRARIES})
+      endif ()
+    endif ()
```

Comment:
is there a way to tell if the module was compiled with mpc/mpfr/boost support ?

Reply:
I propose to add the following in SpecFunc:
```
  static Bool IsBoostAvailable();
  static Bool IsMPFRAvailable();
  static Bool IsMPCAvailable();
```

Reply:
done